src/simplify_reshapes.cpp

```diff
@@ -26,7 +26,6 @@ const auto& reshaper_names()
     static const std::unordered_set<std::string> names = {
         "flatten",
         "reshape",
-        "contiguous",
```

Comment:
We should still remove redundant contiguous in simplify_reshapes, but only when this doesn't precede an operator that requires standard.